src/sentry/deletions/defaults/group.py

```diff
@@ -206,24 +143,6 @@ class ErrorEventsDeletionTask(EventsBaseDeletionTask):
 
     dataset = Dataset.Events
 
-    # XXX: To be removed
-    def post_delete_events_from_nodestore(self, events: Sequence[Event]) -> None:
-        self.delete_dangling_attachments_and_user_reports(events)
-
-    # XXX: To be removed
-    def delete_dangling_attachments_and_user_reports(self, events: Sequence[Event]) -> None:
```

Comment:
This is handled inside of the new task:
https://github.com/getsentry/sentry/blob/cb531042e4cbfad9581f90be569862f5b58efeaf/src/sentry/deletions/tasks/nodestore.py#L114